support/index.html

```diff
@@ -16,44 +16,25 @@ <h1>Support</h1>
                 <div class="card card-support">
                     <div class="card-section">
                         <i class="border-red icon-red fa fa-comments-o fa-2x"></i>
-                        <h4>Forum</h4>
-                        <p class="card-caption">The Scientific Community Image Forum allows the whole community to share their expertise and solutions.</p>
-                        <a href="https://www.openmicroscopy.org/forums" target="_blank" class="btn-blue button small">Visit the Forum</a>
+                        <h4>Image.sc Forum</h4>
+                        <p class="card-caption">
+                            Please use "ome", "omero" or "bio-formats" tags on the Image.sc Forum when posting or
+                            searching.
+                          </p>
+                        <a href="https://forum.image.sc/" target="_blank" class="btn-blue button small">Visit the Forum</a>
```

Comment:
:+1: for updating the name and pointing to tags, but the URL was specifically chosen so we can redirect in the future.